<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update</class>
 <widget class="QMainWindow" name="update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(69, 92, 182, 255), stop:1 rgba(230, 202, 232, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(238, 238, 236);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="batchend">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(185, 22, 226);
background-color: rgb(229, 164, 244);</string>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="text">
              <string>Batch End</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inp_batend">
             <property name="styleSheet">
              <string notr="true">color: rgb(136, 138, 133);
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>enter......</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="batchstart">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(229, 164, 244);</string>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="text">
              <string>Batch Start</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inp_batstart">
             <property name="styleSheet">
              <string notr="true">color: rgb(136, 138, 133);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>enter......</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="Rollno">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(229, 164, 244);</string>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="text">
              <string>Roll No.</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inp_rollno">
             <property name="styleSheet">
              <string notr="true">color: rgb(136, 138, 133);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>enter......</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="class_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(229, 164, 244);</string>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="text">
              <string>Class</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inp_class">
             <property name="styleSheet">
              <string notr="true">color: rgb(136, 138, 133);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>enter......</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="name">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(229, 164, 244);</string>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inp_name">
             <property name="styleSheet">
              <string notr="true">color: rgb(136, 138, 133);</string>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>enter......</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="SHOW">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>44</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Umpush</family>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(114, 31, 203);
color: rgb(238, 238, 236);</string>
           </property>
           <property name="text">
            <string>SHOW</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="CAPTURE">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>44</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Umpush</family>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(114, 31, 203);
color: rgb(238, 238, 236);</string>
           </property>
           <property name="text">
            <string>CAPTURE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="imgLabel">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>7</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="BACK">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Umpush</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(114, 31, 203);
color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>BACK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="UPDATE_IT">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Umpush</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(114, 31, 203);
color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>UPDATE</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
